jekyll/_cci2/stop-building-a-project-on-circleci.adoc

```diff
@@ -21,7 +21,7 @@ This page outlines how to stop builds for a project on CircleCI, and how to dele
 
 CAUTION: The **Stop Building** option will not automatically remove all project data from the CircleCI databases. To remove all project data from CircleCI you need to contact CircleCI support, as detailed in the <<remove-a-project-from-circleci>> section.
 
-NOTE: The **Stop Building** option is supported for GitHub OAuth and Bitbucket Cloud integrations with CircleCI. **Stop Building** is not supported for GitLab, GitHub App or Bitbucket Data Center projects. To find out if you authorized through the GitHub OAuth app or the CircleCI GitHub App, see the xref:github-apps-integration#[GitHub App integration] page.
+include::../_includes/partials/notes/standalone-unsupported.adoc[This feature is not supported for GitLab, GitHub App or Bitbucket Data Center]
```

Comment:
I want to double check if "stop building" applies to hybrid projects

Reply:
No need to update here for now, but I have my answer. 
"Stop building":
- Stops building OAuth pipelines
- Does not stop building GitHub App pipelines (if you have triggers set up, of course)
- makes it so your project does not appear in the "pipelines" page, so it's difficult to track any GitHub App pipelines that may still be running